op-chain-ops/deployer/init.go

```diff
@@ -75,10 +75,15 @@ func Init(cfg InitConfig) error {
 	}
 
 	intent := &state.Intent{
-		L1ChainID:            cfg.L1ChainID,
-		FundDevAccounts:      true,
-		ContractsRelease:     "op-contracts/v1.6.0",
-		ContractArtifactsURL: V160ArtifactsURL,
+		L1ChainID:                  cfg.L1ChainID,
+		FundDevAccounts:            true,
+		ContractsRelease:           "op-contracts/v1.6.0",
+		ContractArtifactsURL:       V160ArtifactsURL,
+		BaseFeeVaultRecipient:      common.Address{},
+		L1FeeVaultRecipient:        common.Address{},
+		SequencerFeeVaultRecipient: common.Address{},
+		Eip1559Denominator:         50,
+		Eip1559Elasticity:          6,
```

Comment:
Looks like the new values here are in the root config, like this:
```toml
l1ChainID = 11155111
fundDevAccounts = true
contractArtifactsURL = "https://storage.googleapis.com/oplabs-contract-artifacts/artifacts-v1-ee07c78c3d8d4cd8f7a933c050f5afeebaa281b57b226cc6f092b19de2a8d61f.tar.gz"
contractsRelease = "op-contracts/v1.6.0"
baseFeeVaultRecipient = "0x0000000000000000000000000000000000000000"
l1FeeVaultRecipient = "0x0000000000000000000000000000000000000000"
sequencerFeeVaultRecipient = "0x0000000000000000000000000000000000000000"
eip1559Denominator = 50
eip1559Elasticity = 6

[[chains]]
  id = "0x0000000000000000000000000000000000000000000000000000000000000002"
  [chains.roles]
    proxyAdminOwner = "0xc2681254d6b08b1eebfd841d75bdaf581c14f810"
    systemConfigOwner = "0x3e4f624ddf5572b89b051f6773ea044582d2f207"
    governanceTokenOwner = "0xc2681254d6b08b1eebfd841d75bdaf581c14f810"
    unsafeBlockSigner = "0x39db188b2058d54016a0ee49dcc59781b5cd67d8"
    batcher = "0x8d394e2ddb85074f395713b613a4d896af3aa898"
    proposer = "0xed5f4989bab532801afff8d629eedd32326ef91d"
    challenger = "0x4ffa5cb5b17e6587821ccf01491bfabc3376675a"
```

but we should make them chain specific, like this, because each chain may use different values for these

l1ChainID = 11155111
fundDevAccounts = true
contractArtifactsURL = "https://storage.googleapis.com/oplabs-contract-artifacts/artifacts-v1-ee07c78c3d8d4cd8f7a933c050f5afeebaa281b57b226cc6f092b19de2a8d61f.tar.gz"
contractsRelease = "op-contracts/v1.6.0"

[[chains]]
  id = "0x0000000000000000000000000000000000000000000000000000000000000002"
  baseFeeVaultRecipient = "0x0000000000000000000000000000000000000000"
  l1FeeVaultRecipient = "0x0000000000000000000000000000000000000000"
  sequencerFeeVaultRecipient = "0x0000000000000000000000000000000000000000"
  eip1559Denominator = 50
  eip1559Elasticity = 6
  [chains.roles]
    proxyAdminOwner = "0xc2681254d6b08b1eebfd841d75bdaf581c14f810"
    systemConfigOwner = "0x3e4f624ddf5572b89b051f6773ea044582d2f207"
    governanceTokenOwner = "0xc2681254d6b08b1eebfd841d75bdaf581c14f810"
    unsafeBlockSigner = "0x39db188b2058d54016a0ee49dcc59781b5cd67d8"
    batcher = "0x8d394e2ddb85074f395713b613a4d896af3aa898"
    proposer = "0xed5f4989bab532801afff8d629eedd32326ef91d"
    challenger = "0x4ffa5cb5b17e6587821ccf01491bfabc3376675a"

Reply:
Changed in: https://github.com/ethereum-optimism/optimism/pull/12404/commits/2d3aa8a87c5131f22242dc85bd8209a4ba47a81d